src/transformers/models/time_series_transformer/modeling_time_series_transformer.py

```diff
@@ -1499,9 +1499,9 @@ def create_network_inputs(
         self,
         past_values: torch.Tensor,
         past_time_features: torch.Tensor,
-        past_observed_mask: torch.Tensor,
         static_categorical_features: torch.Tensor,
         static_real_features: torch.Tensor,
+        past_observed_mask: Optional[torch.Tensor] = None,
```

Comment:
Hmm I see you updated the order here. But I'd like to have past_observed_mask to be after `past_values` and `past_time_values`.

Could you revert this (and not setting past_observed_mask to None by default)?